### DISCUSSION TITLE
编译Tools: Thrift时报错

### DISCUSSION THREAD

**Discussion by Firemountaincold** [apache/iotdb#4652] [Q&A] [answered]
本人java小白，想请教一下这是怎么回事，如何处理，谢谢
![Screenshot 2021-12-29 153832](https://user-images.githubusercontent.com/28642068/147638388-76f004b6-0fdf-4e1e-83ce-0d4402caf0b4.png)

**Comment by HTHou** [comment, marked as answer]
Windows 下的编译 Thrift 的流程有一些问题。
https://github.com/apache/iotdb/pull/4665 可以参考这个PR，手动修改一下pom文件后再试试？

**Reply by Firemountaincold** [reply]
谢谢，编译成功了
![Screenshot 2021-12-30 155554](https://user-images.githubusercontent.com/28642068/147732387-97601dde-54c5-4337-8ea5-c8a5e2a8d542.png)
